src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private static final int DIVIDER_FOR_AREA = 2;
```

Comment:
The constant `DIVIDER_FOR_AREA` is not needed here since the area of a rectangle is calculated by multiplying the length of both sides. Please remove this constant to adhere to the checklist point about magic numbers in code, as it is not a magic number in this context.